dds/DCPS/FilterExpressionGrammar.h

```diff
@@ -83,30 +80,47 @@ namespace FilterExpressionGrammar {
   struct OP_GTEQ : Tok<Seq<Char<'>'>, Char<'='> > > {};
   struct OP_NEQ : Tok<Seq<Char<'<'>, Char<'>'> > > {};
   struct OP_LIKE : Keyword<CharSeqIgnoreCase<'l', 'i', 'k', 'e'> > {};
-  struct RelOp : Or<Store<OP_EQ>, Store<OP_LT>, Store<OP_GT>, Store<OP_LTEQ>,
-    Store<OP_GTEQ>, Store<OP_NEQ>, Store<OP_LIKE> > {};
-
-  struct FieldName : DelimitedList<Seq<Letter, Star<IdentNextChar> >,
-    Char<'.'> > {};
-
-  struct BetweenPred : Seq<ST<FieldName>, Opt<Store<NOT> >, BETWEEN,
-    Param, AND, Param> {};
-  struct CmpFnParam : Seq<ST<FieldName>, RelOp, Param> {};
-  struct CmpParamFn : Seq<Param, RelOp, ST<FieldName> > {};
-  struct CmpBothFn : Seq<ST<FieldName>, RelOp, ST<FieldName> > {};
-  struct Pred : Or<Store<BetweenPred>, Store<CmpFnParam>, Store<CmpParamFn>,
-    Store<CmpBothFn> > {};
+  struct RelOp : Or<
+    Store<OP_EQ>,
+    Store<OP_LT>,
+    Store<OP_GT>,
+    Store<OP_LTEQ>,
+    Store<OP_GTEQ>,
+    Store<OP_NEQ>,
+    Store<OP_LIKE>
+    > {};
+
+  struct FieldName : DelimitedList<Seq<Letter, Star<IdentNextChar> >, Char<'.'> > {};
+
+  struct Primary;
+  struct _Call : Seq< ST<FieldName>, Opt<Seq<Tok<Char<'('> >, Primary, Tok<Char<','> >, Primary, Tok<Char<')'> > > > > { };
```

Comment:
Pick a different naming convention so we don't have identifiers starting with _ followed by a capital letter.